yt/python/yt/environment/components/query_tracker/__init__.py

```diff
@@ -63,11 +63,16 @@ def run(self):
             "stages": {
                 "production": {
                     "channel": {
+                        # NB(mpereskokova): Required for the bans mechanics;
+                        # In case of disabled balancing on single address discovery,
+                        # discovery requests are not sent and bans are not executed
+                        "disable_balancing_on_single_address": False,
```

Comment:
Should we do the same in ytsaurus-k8s-operator?

Reply:
Yes, after finishing this PR I will do it